packages/plugin-core/src/components/lookup/NotePickerUtils.ts

```diff
@@ -0,0 +1,349 @@
+/* eslint-disable no-dupe-class-members */
+import {
+  DendronError,
+  DEngineClient,
+  DNodeProps,
+  DNodePropsQuickInputV2,
+  DNodeUtils,
+  NoteLookupUtils,
+  NoteProps,
+  NoteQuickInput,
+  SchemaUtils,
+} from "@dendronhq/common-all";
+import { DLogger, getDurationMilliseconds } from "@dendronhq/common-server";
+import { HistoryService, LinkUtils } from "@dendronhq/engine-server";
+import _ from "lodash";
+import path from "path";
+import { ExtensionProvider } from "../../ExtensionProvider";
+import { Logger } from "../../logger";
+import { VSCodeUtils } from "../../vsCodeUtils";
+import { getDWorkspace } from "../../workspace";
```

Comment:
do these touch on too many places to swap out in this PR? if so we should take note of it and clean it up on the next round of cleanup.

Reply:
Nope - just missed it.  Cleaned up in latest commit.